### DISCUSSION TITLE
KEYCLOAK_USER / KEYCLOAK_ADMIN / KEYCLOAK_PASSWORD / KEYCLOAK_ADMIN_PASSWORD

### DISCUSSION THREAD

**Discussion by lgh06** [keycloak/keycloak#10003] [Q&A]
I am confused by  https://github.com/keycloak/keycloak-containers/blob/main/server/README.md  and  https://github.com/keycloak/keycloak-web/blob/main/guides/includes/getting-started/start-keycloak-container.adoc  .   

both of them set `-e` , they are not aligned to same.

which environment var should i use...?  
or any explanations ?  
thanks.

**Comment by oleks-popovych** [comment]
This command worked for me, it created default realm master with this `admin` default user
```
docker run  --rm  -p 8080:8080 \
  -e KEYCLOAK_USER=admin \
  -e KEYCLOAK_PASSWORD=admin \
  jboss/keycloak
```
